src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -754,7 +754,7 @@ internal static bool IsSelfSigned(X509Chain chain)
 
         internal static AsymmetricAlgorithm GetAnyPublicKey(X509Certificate2 certificate)
         {
-            return (AsymmetricAlgorithm)certificate.GetRSAPublicKey() ?? certificate.GetDSAPublicKey();
+            return (AsymmetricAlgorithm)certificate.GetRSAPublicKey();
```

Comment:
The only way I know around it is to also make a netcoreapp build from `src`.  Or there's reflection, which may make aot upset.  I'm a bit if there are no tests that hit it... but given that it's DSA I'm not entirely surprised.

Provided we can describe what loss of functionality this entails and we document it, I'd be okay with it (https://github.com/dotnet/corefx/wiki/ApiCompat).